lib/compare.js

```diff
@@ -26,7 +26,6 @@ const equalObject = (obj1, obj2) => {
 
 const nativeFunctionPattern = /^function \w*\(\) \{ \[native code\] \}$/;
 const equalFunction = (func1, func2) => {
-  if (func1 === null || func2 === null) return func1 === func2;
```

Comment:
Please explain this

Reply:
The problem was described here https://github.com/metarhia/metatests/pull/144#issuecomment-461111201